client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -352,6 +355,7 @@ export const PrescriptionLineEditForm: React.FC<
                       },
                     },
                   }}
+                  onBlur={() => abbreviationRef.current?.focus()}
```

Comment:
I don't suppose there is a cleaner way to get "Tab" from the quantity input to focus here? Cos by doing it with `onBlur`, this steals focus regardless of whether you Tabbed or not -- if I intentionally click in the "Presribed Quantity" field (for example) cos I want to change it, I get pulled to the "Abbreviation" field which is not a great user experience.

Reply:
thanks - that's a good point. Have changed to watch for a `Tab` keypress